source/CIGameScene.cpp

```diff
@@ -114,6 +129,7 @@ bool GameScene::init(const std::shared_ptr<cugl::AssetManager>& assets,
     addChild(scene);
     addChild(_planet->getPlanetNode());
     addChild(_stardustContainer->getStardustNode());
+    addChild(_pauseMenu->getLayer(), 1);
```

Comment:
I don't think we need to store layer in the pause menu class. I think you should just directly add the child the same way scene is added. Storing the layer field is a little weird imo

Reply:
we need to get the layer to convert from menu to scene node, otherwise it errors out

Reply:
_layer = assets->get<scene2::SceneNode>("pause");
_layer->setContentSize(dimen);
 _layer->doLayout();
 
 Move this to game scene and you should be able to

Reply:
I thought we only wanted one scene in each class, which is why I separated into PauseMenu.
GameScene already has `scene`, should we add `pausemenu` to that as well?